src/utils/colors.ts

```diff
@@ -8,7 +8,7 @@ import {
 	getScale,
 } from 'color2k';
 
-const isValidColor = color => {
+const isValidColor = (color: string): [number, number, number, number] | boolean => {
```

Comment:
This will only ever return a Boolean, the result of `parseToRgba` isn't returned.
```suggestion
const isValidColor = (color: string): boolean => {
```

Reply:
made change